launchable/utils/http_client.py

```diff
@@ -31,7 +31,7 @@ def request(self, method, sub_path, payload=None, timeout=(5, 60), compress=Fals
 
         headers = self._headers(compress)
 
-        Logger().audit(AUDIT_LOG_FORMAT.format("post", url, headers, payload))
```

Comment:
The method should be `method`, not `"post"`. This is a bug I incorporated in my previous PR...